packages/core/src/validate.js

```diff
@@ -263,25 +263,17 @@ export default function validateFormData(
   };
 }
 
-function isDict(v) {
-  return (
-    typeof v === "object" &&
-    v !== null &&
-    !(v instanceof Array) &&
-    !(v instanceof Date)
-  );
-}
-
 /**
  * Get a similar schema where ref's are prefixed with "__rjsf_rootSchema"
+ * This is used in isValid to make references to the rootSchema
  */
 function withIdRefPrefix(schema) {
```

Comment:
Will this function work properly with a schema that has an array of objects, one of which has a $ref?

Reply:
I guess you're right. We should have a test for that